spec/schema.json

```diff
@@ -2,11 +2,11 @@
   "swagger": "2.0",
   "info": {
     "version": "1.0.0",
-    "title": "LFP PAY API",
-    "description": "An API that creates, patches, and retrieves LFP sessions, as well as interacting with the E5 finance system"
+    "title": "PPS PAY API",
```

Comment:
Are we able to make this match what we have in the README too please - `Penalty Payment Service (PPS) API`?

Reply:
Yeah, can update